middleware/feed.ts

```diff
@@ -0,0 +1,8 @@
+export default defineNuxtRouteMiddleware((from) => {
+  if (!from.params.feed || !validFeeds.includes(from.params.feed as string)) {
+    return abortNavigation('not found')
```

Comment:
for now, can we redirect to a safe path instead?

Reply:
Oh indeed! I made the changes yesterday but failed to push and didn't notice 😅